### ISSUE TITLE
Thanks for all the good work!

### ISSUE THREAD

**Issue by chsull** [brbeaird/SmartThings-MyQ-Edge#51]
As a long-time user, I want to thank you for your efforts — much appreciated! Chamberlain is scratched completely off my list of companies I’d do business with again…

**Comment by brbeaird** [comment]
I appreciate it. Glad I could offer something useful for as long as it was! Maybe I can do something similar with RatGdo for anyone interested.

**Comment by gnieboer** [comment]
Yep, just adding in a thank you to the list, the work was much appreciated!
